src/main/resources/db/migration/V24__iat_ddl.sql

```diff
@@ -0,0 +1,34 @@
+DO $$
+DECLARE
+    roleId integer;
+    permissionId integer;
+    createdBy integer;
+BEGIN
+    SELECT COALESCE(MIN(id), 1) FROM tims_role INTO roleId;
+    SELECT id FROM tims_permission WHERE code = 'can_edit_sa_text_editor_type' INTO permissionId;
+    SELECT COALESCE(MIN(id), 1) FROM tims_user INTO createdBy;
+
+    UPDATE
+        tims_permission
+    SET
+        is_workflow_restricted = true
+    WHERE
+        id = permissionId;
+
+    INSERT INTO tims_role_permission_workflow_status(
```

Comment:
@jeffjohnson9046  We have not added data to the `tims_role_permission_workflow_status` table via migrations. We created a `sample-seed.sql` to add data via a Postgres client only to QA to facilitate front-end testing. Smarter Balance did not provide any pre-configured roles that need to be created in production.